api/src/main/resources/messages.properties

```diff
@@ -0,0 +1,2 @@
+${project.parent.artifactId}.title=patientflag_rest Module
```

Comment:
Is this file used/required? Can we using the [existing one](https://github.com/openmrs/openmrs-module-patientflags/blob/master/omod/src/main/resources/messages.properties) instead?